test/performance/broker-latency/broker-latency-setup.yaml

```diff
@@ -76,7 +76,7 @@ metadata:
   name: perf-eventing
 rules:
   - apiGroups: [""]
-    resources: ["nodes"]
+    resources: ["nodes", "pods"]
```

Comment:
Yep. senders-receivers wait for all Pods to be Running before starting the benchmark because it wouldn't make sense to start before the IMC channel and other CE receivers are up and running.